src/status_im/contexts/wallet/account/share_address/view.cljs

```diff
@@ -50,7 +50,7 @@
 (defn view
   []
   (let [padding-top         (:top (safe-area/get-insets))
-        wallet-type         (reagent/atom :legacy)
+        wallet-type         (reagent/atom :multichain)
```

Comment:
Shouldn't this be a react state according to the new ui guideline? https://github.com/status-im/status-mobile/blob/develop/doc/ui-guidelines.md?plain=1#L11-L44

Reply:
I mean yes and no. This pr is not a refactor pr, as that also leaves rooms for bugs to sneak in. I just want to make the smallest change possible here which is to make this ui right in terms of order of the tabs. New code as a defult should be written with react state ofc 👍